### PR TITLE
Add SF engineering role

### PR DESCRIPTION
**Pull request by tsolakoua**
This is the description of the Applications role in SF, after the requirements I received last Friday from @eiso and @campoy. 

After receiving your feedback, I will work on the senior PM that should be very similar to this one. 

I didn't add "Interaction with eventual/ potential customers" responsibility and will add it in the PM role, as I think it's more relevant and will also add some difference to the roles that look very similar. 

Tech-wise it might seem a bit broad as I didn't mention anywhere the technologies he needs to know. Please update me on that in case it's not the same with the Applications requirements we have in Madrid  role (You will be expected to have strong backend coding skills in at least one programming language and very good algorithmic skills. Scala coding skills and knowledge about Apache Spark aren't required but will be highly appreciated).

### CODE REVIEW

File: talent/job-descriptions/applications_eng_SF.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+- [bblfsh/dashboard](https://github.com/bblfsh/dashboard): a playground application for Babelfish server, turning code files into Universal Abstract Syntax Trees (UASTs). 
+- [src-d/kmcuda](https://github.com/src-d/kmcuda):  fast K-means and K-nearest neighbours on NVIDIA GPUs.  
+- [src-d/minhashcuda](https://github.com/src-d/minhashcuda):  fast Weighted MinHash on NVIDIA GPUs. It allows for fast nearly-duplicate repository detection: scan +17 git repositories in 30 minutes to find 1.5M duplicates (the results were published on data.world).  
+- [src-d/engine](https://github.com/src-d/engine):  a library for running scalable data retrieval pipelines that process any number of Git repositories for source code analysis.  
+- [src-d/hercules](https://github.com/src-d/hercules):  analysis and visualization of a Git repository history. 
+- [src-d/gemini](https://github.com/src-d/gemini): a duplicate finder for source code in git repositories.
+
+## ROLE
+
+This position is for a Senior Engineer on the Applications team, based in SF or remote.
```
Comment (bzz) [inline talent/job-descriptions/applications_eng_SF.md]: Was under impression that this would be a position only for SF/Valley.

**Comment by bzz** [comment]
From my side
 - On `Interaction with eventual/ potential customers` - I thought that this was the purpose of having this position in SF

 - would be interesting to know more on what
   > .. the senior PM that should be very similar to this one.
  
   means

File: talent/job-descriptions/applications_eng_SF.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+- [src-d/minhashcuda](https://github.com/src-d/minhashcuda):  fast Weighted MinHash on NVIDIA GPUs. It allows for fast nearly-duplicate repository detection: scan +17 git repositories in 30 minutes to find 1.5M duplicates (the results were published on data.world).  
+- [src-d/engine](https://github.com/src-d/engine):  a library for running scalable data retrieval pipelines that process any number of Git repositories for source code analysis.  
+- [src-d/hercules](https://github.com/src-d/hercules):  analysis and visualization of a Git repository history. 
+- [src-d/gemini](https://github.com/src-d/gemini): a duplicate finder for source code in git repositories.
+
+## ROLE
+
+This position is for a Senior Engineer on the Applications team, based in SF or remote. 
+
+- You will be expected to have a deep understanding and previous experience in developer tooling on a large scale.
```
Comment (bzz) [inline talent/job-descriptions/applications_eng_SF.md]: I would suggest being more specific by adding something like  `2+ years of previous experience working on dev tooling...`

File: talent/job-descriptions/applications_eng_SF.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+- [src-d/hercules](https://github.com/src-d/hercules):  analysis and visualization of a Git repository history. 
+- [src-d/gemini](https://github.com/src-d/gemini): a duplicate finder for source code in git repositories.
+
+## ROLE
+
+This position is for a Senior Engineer on the Applications team, based in SF or remote. 
+
+- You will be expected to have a deep understanding and previous experience in developer tooling on a large scale. 
+
+- Your main responsibilities will be prototyping and explore the future of developer tooling by discovering new applications that could be meaningful for the developer community.
```
Comment (campoy) [inline talent/job-descriptions/applications_eng_SF.md]: prototyping and *exploring*

Comment (campoy) [reply talent/job-descriptions/applications_eng_SF.md]: for the developer community -> to the developer community

File: talent/job-descriptions/applications_eng_SF.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+## ROLE
+
+This position is for a Senior Engineer on the Applications team, based in SF or remote. 
+
+- You will be expected to have a deep understanding and previous experience in developer tooling on a large scale. 
+
+- Your main responsibilities will be prototyping and explore the future of developer tooling by discovering new applications that could be meaningful for the developer community. 
+
+- We are looking for someone who has a decent understanding of Machine Learning, as you will deal with models built using modern ML stacks written in Python and C++ like Tensorflow, Tensorboard and CUDA.
```
Comment (campoy) [inline talent/job-descriptions/applications_eng_SF.md]: decent -> good

File: talent/job-descriptions/applications_eng_SF.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+- You will be working closely with the Developer Relations team in order to explore and identify ideas. You will have the opportunity to write blog posts and make videos about our technology, even though it's not main responsibility of the role.
```
Comment (campoy) [inline talent/job-descriptions/applications_eng_SF.md]: not main -> not the main

File: talent/job-descriptions/applications_eng_SF.md (hunk 3 above)
Comment (tsolakoua) [reply talent/job-descriptions/applications_eng_SF.md]: Blaming Grammarly for the "explore" xD

**Comment by tsolakoua** [comment]
@eiso Regarding the above comment of @bzz  about the senior PM role, I understood from our discussion that it will be very similar to the dev role, adding in the description a couple of additional responsibilities that PMs have. Is that correct?

**Comment by campoy** [comment]
I wouldn't have merged yet, @eiso should approve too

**Comment by tsolakoua** [comment]
Sorry @campoy, I reverted it. I thought approval from one of you two would be suitable.

**Pull request merged**